apps/cookbook/src/app/examples/grid-layout-example/grid-layout-extended-example/grid-layout-extended-example.component.scss

```diff
@@ -1,29 +1,29 @@
 @import './libs/designsystem/src/lib/scss/utils';
 
-// Grid
+/* Configure grid properties */
 $columns: 12;
 $gap: size('m');
 
-// Core example
+/* Declare the grid container */
 .grid-container {
   display: grid;
   grid-template-columns: repeat($columns, 1fr);
   gap: $gap;
 }
 
-// Core example
+/* Let grid items span all columns by default */
 .grid-item {
   grid-column: span $columns;
 }
 
-// Core example
+/* Tablet size and above */
 @include media('>=medium') {
   .half\@tablet-up {
```

Comment:
General statement for all these classes with `\@something`. 

I actually like the naming, but seeing it in the code snippets in the cookbook, I think we could risk that people will interpret this as some kind of special notation.

Could we do `half-at-tablet-up`?

Reply:
I agree. The naming is not relevant so the class selectors could be whatever. Let's go with `half-at-tablet-up` etc.